spec/fixtures/tags

```diff
@@ -4,7 +4,7 @@
 !_TAG_PROGRAM_NAME	Exuberant Ctags	//
 !_TAG_PROGRAM_URL	http://ctags.sourceforge.net	/official site/
 !_TAG_PROGRAM_VERSION	5.8	//
-callMeMaybe	tagged.js	/^function callMeMaybe() {$/;"	f
-duplicate	tagged-duplicate.js	/^function duplicate() {$/;"	f
-duplicate	tagged.js	/^function duplicate() {$/;"	f
-thisIsCrazy	tagged.js	/^var thisIsCrazy = true;$/;"	v
+callMeMaybe	tagged.js	/^function callMeMaybe() {$/;"	f	line:3
```

Comment:
It would be good to have a separate tags file without line numbers and add some spec coverage for what `lineNumber` returns when the value isn't present in the tags file.